lightstep-tracer-jre/src/test/java/com/lightstep/tracer/jre/JRETracerTest.java

```diff
@@ -139,7 +139,7 @@ public void spanBuilderWithTagTest() throws Exception {
         Span span = tracer
                 .buildSpan("test_span")
                 .withTag("my_key", "my_value")
-                .start();
+                .startManual();
         span.finish();
```

Comment:
for all of the test spans that are actually finished, please replace with a try-with-resources block and `startActive`